common/app/views/support/Profile.scala

```diff
@@ -36,13 +38,21 @@ sealed trait ElementProfile {
 
   // NOTE - if you modify this in any way there is a decent chance that you decache all our images :(
   lazy val resizeString = {
-    val qualityparam = "q=85"
+    val qualityparam = if (hidpi) {"q=20"} else {"q=55"}
```

Comment:
apparently imgix are looking to add lossy compression for PNGs which we would automatically pick up if it's left on them. think will leave it, since there's no specific benefit to removing it